src/SFA.DAS.AssessorService.Api.Types/Models/Validation/ValidationRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿using MediatR;
+
+namespace SFA.DAS.AssessorService.Api.Types.Models.Validation
+{
+    public class ValidationRequest : IRequest<bool>
+    {
+
+
+        public string ValidationType { get; set; }
+        public string ValidationString { get; set; }
+        public string ValidationMatchValue { get; set; }
```

Comment:
Maybe 

`public string Type { get; set; }

        public string String { get; set; }

        public string MatchValue { get; set; }`  

we know they are Validation related. 👍 ❓

Reply:
sounds legit.  Not calling a field 'String' though!  I Think ValueToCheck maybe...